src/main/kotlin/no/nav/dagpenger/regel/api/MinsteinntektBeregninger.kt

```diff
@@ -0,0 +1,25 @@
+package no.nav.dagpenger.regel.api
+
+import java.lang.Exception
+
+data class MinsteinntektBeregningResultat(
+    val oppfyllerMinsteinntekt: Boolean,
+    val periode: Int
+) {
+    companion object {
+        val exampleInntektBeregning = mapOf(
+            "oppfyllerMinsteinntekt" to true,
+            "status" to 1
+        )
+    }
+}
+
+class MinsteinntektBeregninger {
+    val beregninger = mutableMapOf<String, MinsteinntektBeregningResultat>(
+        "123" to MinsteinntektBeregningResultat(true, 52)
+    )
+
+    fun hasDataForBeregning(beregningsId: String) = beregninger.containsKey(beregningsId)
+
+    fun getBeregning(beregningsId: String) = beregninger[beregningsId] ?: throw Exception("no beregning for id found")
```

Comment:
Do not use raw exception types - I think it is better to create our own or reuse a specific exception if possible. e-g. BeregningNotFoundException